docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+# Release 352 (XX YYY 2020)
+
+## General
+
+* Add support for [`WINDOW` clause](window_clause). ({issue}`651`)
+* Add {func}`version` function. ({issue}`4627`)
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to been equal. ({issue}`6472`) 
+* Fix query failure when using element of `array(timestamp(p))` in a complex expression 
+  for `p` greater than 6. ({issue}`6350`)
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
+* Add scopes to `OAuth2` configuration. ({issue}`6580`)
+* Prevent from replay attacks in `OAuth2` by using `nonce` cookie when `openid` scope is requested. ({issue}`6580`)
+
+## JDBC Driver
+
+* Support user impersonation when using password-based authentication. This is configured 
+  using `sessionUser` connection URL parameter. Previously user impersonation was possible
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support.({issue}`6576`)
+
+## Docker Image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery Connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
```

Comment:
"for views" -> "on a view"? or maube

"Fix query failure when calculating `count(*)` aggregation on a view, without any filter."

Reply:
Sorry, I should have mentioned that this issue doesn't happen for the first call. Maybe, 
"Fix query failure when calculating count(*) aggregation on a view more than once, without any filter." or something.